docs/code/round_3/pages.py

```diff
@@ -21,7 +21,7 @@ def search_results(self):
         """
         Return a list of results returned from a search
         """
-        return self.q(css='ul.repo-list > li > div > h3 > a').text
+        return self.q(css='ul.repo-list > li > div > div > a').text
```

Comment:
@morenol whats a purpose of this change?

Reply:
Ok fixing a test.

Reply:
To make the tests pass, Github changed the style of that page that is used in the test so I had to change that.